openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -93,6 +93,35 @@
       "description": "Regex to only fetch databases that matches the pattern.",
       "$ref": "../type/filterPattern.json#/definitions/filterPattern",
       "title": "Database Filter Pattern"
+    },
+    "incremental": {
+      "description": "Use incremental Metadata extraction after the first execution. This is commonly done by getting the changes from Audit tables on the supporting databases.",
+      "type": "object",
+      "title": "Incremental Metadata Extraction Configuration",
+      "properties": {
```

Comment:
instead of having the properties directly here, maybe we can use a `definition` on the same file